tracee-ebpf/tracee/events_config.go

```diff
@@ -91,488 +90,3850 @@ const (
-	UtimensatTime32EventID:        {ID: UtimensatTime32EventID, ID32Bit: sys32utimensat, Name: "utimensat_time32", Probes: []probe{{event: "utimensat_time32", attach: sysCall, fn: "utimensat_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	Pselect6Time32EventID:         {ID: Pselect6Time32EventID, ID32Bit: sys32pselect6, Name: "pselect6_time32", Probes: []probe{{event: "pselect6_time32", attach: sysCall, fn: "pselect6_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	PpollTime32EventID:            {ID: PpollTime32EventID, ID32Bit: sys32ppoll, Name: "ppoll_time32", Probes: []probe{{event: "ppoll_time32", attach: sysCall, fn: "ppoll_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	IoPgeteventsTime32EventID:     {ID: IoPgeteventsTime32EventID, ID32Bit: sys32io_pgetevents, Name: "io_pgetevents_time32", Probes: []probe{{event: "io_pgetevents_time32", attach: sysCall, fn: "io_pgetevents_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	RecvmmsgTime32EventID:         {ID: RecvmmsgTime32EventID, ID32Bit: sys32recvmmsg, Name: "recvmmsg_time32", Probes: []probe{{event: "recvmmsg_time32", attach: sysCall, fn: "recvmmsg_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	MqTimedsendTime32EventID:      {ID: MqTimedsendTime32EventID, ID32Bit: sys32mq_timedsend, Name: "mq_timedsend_time32", Probes: []probe{{event: "mq_timedsend_time32", attach: sysCall, fn: "mq_timedsend_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	MqTimedreceiveTime32EventID:   {ID: MqTimedreceiveTime32EventID, ID32Bit: sys32mq_timedreceive, Name: "mq_timedreceive_time32", Probes: []probe{{event: "mq_timedreceive_time32", attach: sysCall, fn: "mq_timedreceive_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	RtSigtimedwaitTime32EventID:   {ID: RtSigtimedwaitTime32EventID, ID32Bit: sys32rt_sigtimedwait, Name: "rt_sigtimedwait_time32", Probes: []probe{{event: "rt_sigtimedwait_time32", attach: sysCall, fn: "rt_sigtimedwait_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	FutexTime32EventID:            {ID: FutexTime32EventID, ID32Bit: sys32futex, Name: "futex_time32", Probes: []probe{{event: "futex_time32", attach: sysCall, fn: "futex_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	SchedRrGetInterval32EventID:   {ID: SchedRrGetInterval32EventID, ID32Bit: sys32sched_rr_get_interval, Name: "sched_rr_get_interval_time32", Probes: []probe{{event: "sched_rr_get_interval_time32", attach: sysCall, fn: "sched_rr_get_interval_time32"}}, Sets: []string{"syscalls", "32bit_unique"}},
-	HiddenInodesEventID:           {ID: HiddenInodesEventID, ID32Bit: sys32undefined, Name: "hidden_inodes", Probes: []probe{{event: "filldir64", attach: kprobe, fn: "trace_filldir64"}}, Sets: []string{}},
+var EventsConfig = map[int32]EventConfig{
```

Comment:
Looks great. 
The map is a very big one. What about extracting it to a json file and read it? the advantages of doing so is to have a more ordered code, the events_config.co file will be smaller - if more components need this map they will just have to read the json and so splitting the code in packages will be easier as there will be less dependencies.

Reply:
I don't think that the fact that we do it in go is less ordered than json.
This map can be exported (if we will see that we need this) as a go struct, saving the need to parse a json file.
For now, I don't think it is an advantage to convert this map into json.
If we will decide it is required, this PR is a step forward anyways in that direction.